test/e2e/clustertask/start_test.go

```diff
@@ -121,4 +132,95 @@ Waiting for logs to be available...
 				return nil
 			}})
 	})
+
+	t.Run("Start ClusterTask with --use-param-defaults and all the params having default", func(t *testing.T) {
+		res := tkn.MustSucceed(t, "clustertask", "start", "read-clustertask",
+			"-i=source="+tePipelineGitResourceName,
+			"-p=FILENAME=README.md",
+			"-w=name=shared-workspace,emptyDir=",
+			"--use-param-defaults",
+			"--showlog",
+			"true")
+		assert.Assert(t, is.Regexp("TaskRun started:.*", res.Stdout()))
+	})
+
+	t.Run("Start ClusterTask with --use-param-defaults and some of the params not having default", func(t *testing.T) {
+		tkn.RunInteractiveTests(t, &cli.Prompt{
+			CmdArgs: []string{"clustertask", "start", "read-clustertask",
+				"-i=source=" + tePipelineGitResourceName,
+				"--use-param-defaults",
+				"-w=name=shared-workspace,emptyDir=",
+				"--showlog",
+				"true"},
```

Comment:
`--showlog` will be `true` without specifying `true`:

```suggestion
				"--showlog"},
```